### PR TITLE
zombienet: BEEFY and MMR test

### PR DESCRIPTION
**Pull request by acatangiu**
Test BEEFY voting and finality.

In the future will test:
- BEEFY RPCs,
- MMR proofs,
- BEEFY block import,
- BEEFY justification syncing.

(Test assumes Rococo sessions of 1 minute)

----------

## Draft status

Waiting for zombienet release containing https://github.com/paritytech/zombienet/pull/276.

### CODE REVIEW

File: zombienet_tests/functional/0003-beefy-and-mmr.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[[relaychain.node_groups]]
+name = "validator"
+count = 3
+args = ["--beefy", "--enable-offchain-indexing=true"]
+
+[types.Header]
+number = "u64"
+parent_hash = "Hash"
+post_state = "Hash"
```
Comment (sandreim) [inline zombienet_tests/functional/0003-beefy-and-mmr.toml:17]: I think you can drop these

**Comment by pepoviola** [comment]
Hi @acatangiu, I just created a new release (https://github.com/paritytech/zombienet/releases/tag/v1.2.50) that contains https://github.com/paritytech/zombienet/pull/276. Thanks!

**Review by pepoviola — APPROVED** [review]
You need to add the test in the gitlab-ci.yml file, I can add you if you want.
Thanks!

**Comment by acatangiu** [comment]
> You need to add the test in the gitlab-ci.yml file, I can add it if you want. Thanks!

Added, please take a look.

**Comment by acatangiu** [comment]
@paritytech-ci please review

**Review by alvicsam — APPROVED** [review]

**Review by radupopa2010 — APPROVED** [review]

**Review by sandreim — APPROVED** [review]
LGTM. I also suggest creating an issue for the leftover todos

**Review by svyatonik — APPROVED** [review]

**Pull request merged**